tests/pds/dataservice_test.go

```diff
@@ -492,6 +492,132 @@ var _ = Describe("{EnableandDisableNamespace}", func() {
 
 })
 
+var _ = Describe("{ScaleDownScaleupPXCluster}", func() {
+	JustBeforeEach(func() {
+		StartTorpedoTest("ScaleDownScaleupPXCluster", "Scales Down the PX cluster, Verify Data Services and Scales up the Px cluster", pdsLabels, 0)
+	})
+
+	It("Scale down PX pods, Verify Data Services and Scale up the PX pods", func() {
+		var generateWorkloads = make(map[string]string)
```

Comment:
Rather multiple times using var keyword , enclose all the variables with types.i.e
var (
x type_x
y type_y
)